scripts/commission.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python3
+import argparse
+import subprocess
+import tempfile
+from time import sleep
+from pathlib import Path
+
+import serial
+from serial.serialutil import SerialException
+
+from insert_serial import pad_serial, insert_bin_serial
+
+BOARD_TYPE = 'PBv4B'
+BOARD_VID = '1bda'
+BOARD_PID = '0010'
+SERIAL_NUM_ADDR = 0x1FE0
+
+
+def dfu_flash_firmware(firmware):
+    subprocess.run(['dfu-util', '-E', '1', '-d', f'{BOARD_VID}:{BOARD_PID}', '-D', firmware])
+
+    sleep(1)
+
+
+def serial_flash_firmware(port, firmware):
+    print(
+        "To flash the board using the factory bootloader you need to connect "
+        "12 volts to the board and connect both the USB and serial ports.")
+    input('Press enter once this is done')
+
+    # # Force board into ROM bootloader by pulling RTS at power on
+    # ser = serial.Serial(port)
+    # ser.rts = False
+    # input('Press the power button on the board twice, then press enter')
+    # ser.close()
+
+    asset_code = input("Enter asset code to bake into the bootloader: ")
```

Comment:
Suggestion: It might be worth explicitly adding to this message that the "sr" prefix is added automatically.